packages/cms-lib/lib/models/OAuth2Manager.js

```diff
@@ -15,6 +15,7 @@ class OAuth2Manager {
       scopes,
       environment = ENVIRONMENTS.PROD,
       tokenInfo = { expiresAt: null, refreshToken: null, accessToken: null },
+      name,
```

Comment:
I am a bit worried that introducing `name` here may have unexpected side effects. For example, it not passed via `fromConfig()`. Are we sure that we have covered all the places where an `OAuth2Manager` is constructed?

Reply:
Sounds like we're covered. Adding unit test coverage would be a way to increase confidence.